environment.yml

```diff
@@ -6,26 +6,27 @@ channels:
   - bioconda
   - defaults
 dependencies:
+ ## conda-forge packages
+  - conda-forge::r-base=3.5.1
   - conda-forge::openjdk=11.0.1 # Needed for FastQC - conda build hangs without this
+  - conda-forge::r-statmod=1.4.30
+  - conda-forge::r-data.table=1.12.2
+  - conda-forge::r-gplots=3.0.1.1
+  - conda-forge::r-r.methodss3=1.7.1
+  - conda-forge::r-markdown=0.9
+  - conda-forge::matplotlib=3.0.3 # Current 3.1.0 build incompatible with multiqc=1.7
+
+  ## bioconda packages
   - fastqc=0.11.8
   - trim-galore=0.6.2
   - samtools=1.9
   - bowtie=1.2.2
   - multiqc=1.7
   - mirtop=0.4.18a
-  - razers3=3.5.3
-  - bioconductor-edger=3.26.0
-  - bioconductor-limma=3.40.0
-  - conda-forge::r-statmod=1.4.30
-  - conda-forge::r-data.table=1.12.2
-  - conda-forge::r-gplots=3.0.1.1
-  - conda-forge::r-r.methodss3=1.7.1
-  - conda-forge::r-markdown=0.9
-  - conda-forge::matplotlib=3.0.3
+  - seqcluster=1.2.5
   - htseq=0.11.2
   - fastx_toolkit=0.0.14
+  - seqkit
```

Comment:
Please pin this to a specific version ;-) https://bioconda.github.io/recipes/seqkit/README.html